spinoso-time/src/time/chrono/build.rs

```diff
@@ -50,6 +51,38 @@ impl Time {
             offset,
         }
     }
+
+    /// Creates a new `Time` object from the `seconds` and `sub_seconds_nanos`
+    /// since the Unix EPOCH with a local offset.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use spinoso_time::Time;
+    /// let epoch = Time::at(0, 0);
+    /// let epoch_plus_1_nano = Time::at(0, 1);
+    /// ```
+    #[inline]
+    #[must_use]
+    pub fn at(seconds: i64, sub_second_nanos: i64) -> Self {
+        let offset = Offset::Local;
+
+        let timestamp = seconds + (sub_second_nanos / i64::from(NANOS_IN_SECOND));
+        let sub_second_nanos = sub_second_nanos % i64::from(NANOS_IN_SECOND);
+
+        // Handle negative sub_seconds_nanos
+        let (timestamp, sub_second_nanos) = if sub_second_nanos > 0 {
+            (timestamp, sub_second_nanos)
+        } else {
+            (timestamp - 1, (i64::from(NANOS_IN_SECOND) - sub_second_nanos.abs()))
+        };
+
+        Self {
+            timestamp,
+            sub_second_nanos: sub_second_nanos as u32,
```

Comment:
let's use a `u32::try_from` conversion here like my suggestion above in the trampoline.